### PR TITLE
Buildkite: remote triggering

### PR DESCRIPTION
**Pull request by leo-ri**
Analog for Jenkins function `triggerRemoteUpdateEPR`

### CODE REVIEW

File: .buildkite/hooks/pre-command [hunk 1]
```diff
@@ -5,9 +5,15 @@ set -euo pipefail
...
 
 DOCKER_REGISTRY_SECRET_PATH="kv/ci-shared/platform-ingest/docker_registry_prod"
-
 if [[ "$BUILDKITE_PIPELINE_SLUG" == "package-registry" && "$BUILDKITE_STEP_KEY" == "publish" ]]; then
-  DOCKER_USERNAME_SECRET=$(retry 5 vault kv get -field user "${DOCKER_REGISTRY_SECRET_PATH}")
-  DOCKER_PASSWORD_SECRET=$(retry 5 vault kv get -field password "${DOCKER_REGISTRY_SECRET_PATH}")
-  docker login -u "${DOCKER_USERNAME_SECRET}" -p "${DOCKER_PASSWORD_SECRET}" "${DOCKER_REGISTRY}" 2>/dev/null
+    DOCKER_USERNAME_SECRET=$(retry 5 vault kv get -field user "${DOCKER_REGISTRY_SECRET_PATH}")
+    DOCKER_PASSWORD_SECRET=$(retry 5 vault kv get -field password "${DOCKER_REGISTRY_SECRET_PATH}")
+    docker login -u "${DOCKER_USERNAME_SECRET}" -p "${DOCKER_PASSWORD_SECRET}" "${DOCKER_REGISTRY}" 2>/dev/null
```
Comment (leo-ri) [inline .buildkite/hooks/pre-command:11]: only spaces: 2 spaces -> 4 spaces

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -74,3 +74,19 @@ steps:
         allow_failure: false
       - step: "lint"
         allow_failure: false
+
+  - label: "Triggering update-package-registry-job-remote with version"
+    key: "triggering-jenkins"
+    if: build.tag =~ /^v\d+\.\d+\.\d+$$/
+    # command: ".buildkite/scripts/trigger-jenkins-job.sh" #TODO uncomment
```
[the remark this answers is not in the document]
Comment (jlind23) [reply .buildkite/pipeline.yml:81]: Does this remote job consider the dry run true option already? If yes, then it is ok,

File: .buildkite/scripts/trigger-jenkins-job.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd .buildkite/scripts/triggerJenkinsJob
+
+go run main.go \
+    --jenkins-job "update-package-registry" \
+    --version "${BUILDKITE_TAG}" \
+    --dry-run "${DRY_RUN}" \
+    --draft "${DRAFT_PR}"
```
[the remark this answers is not in the document]
Comment (mrodm) [reply .buildkite/scripts/trigger-jenkins-job.sh:9]: Were you able to to trigger with dry_run a build of that Jenkins job ?

[the remark this answers is not in the document]
Comment (leo-ri) [reply .buildkite/scripts/trigger-jenkins-job.sh]: @mrodm triggered at a713eaa below test

[the remark this answers is not in the document]
Comment (mrodm) [reply .buildkite/scripts/trigger-jenkins-job.sh]: > @mrodm triggered at [a713eaa](https://github.com/elastic/package-registry/commit/a713eaa70c4e609b9ba73874c64655140db8938e) below (for test)

Perfect! :+1:

**Review by mrodm — APPROVED** [review]

**Pull request merged**